bot/exts/help_channels/_channel.py

```diff
@@ -46,8 +46,19 @@ async def _close_help_post(closed_post: discord.Thread, closing_reason: _stats.C
     """Close the help post and record stats."""
     embed = discord.Embed(description=CLOSED_POST_MSG)
     embed.set_author(name=f"{POST_TITLE} closed", icon_url=CLOSED_POST_ICON_URL)
-
-    await closed_post.send(embed=embed)
+    message = ''
+
+    # Include a ping in the close message if no one else engages, to encourage them
+    # to read the guide for asking better questions
+    if closing_reason == _stats.ClosingReason.INACTIVE and closed_post.owner is not None:
+        participant_ids = {
+            message.author.id async for message in closed_post.history(limit=10, oldest_first=False)
```

Comment:
```suggestion
            message.author.id async for message in closed_post.history(limit=100)
```
100 messages is the max that can be retrieved in 1 request. So we might as well do this to cover the case were the OP sent a lot of messages at the end of a session that does have other participants.

`oldest_first=False` is also the default if `after` isn't specified, so that can be removed.

Reply:
I agree that we should up the limit to 100 if we can get all of that in one request, though I think that keeping `oldest_first=False` is worth it for self-documentation purposes.